app/models/user.rb

```diff
@@ -5,6 +5,7 @@ class User < ActiveRecord::Base
 
   devise :confirmable, :database_authenticatable, :recoverable, :registerable,
          :timeoutable, :trackable, :two_factor_authenticatable, :omniauthable,
+         :zxcvbnable,
```

Comment:
I updated another file in this proj to put a list of attrs like this in a list. What do you think of that? I think it makes it easier to read and alphabetize and see updates in new PRs. Eg:

```
devise(
  :confirmable,
  :database_authenticatable,
  :recoverable,
  :registerable,
  :timeoutable, 
  :trackable,
  :two_factor_authenticatable,
  :omniauthable,
  :zxcvbnable,
)
```